zebra-rpc/src/methods/types/transaction.rs

```diff
@@ -104,6 +104,9 @@ pub struct Output {
     /// The value in zats.
     #[serde(rename = "valueZat")]
     value_zat: i64,
+    /// The value in zats (again with a different name for some reason).
```

Comment:
I left this one out because it looks like an error in zcashd. Seems like they added one variant and then needed the other one and didn't wanted to break the API. i think we can take the opportunity to remove one of them here. 

I added this item for the zcashd deprecation meeting of today.

Reply:
From the meeting today, `valueSat` is going to be deprecated in zcashd 6.2.0 or in a future release, so we better just don't add it to Zebra.

Reply:
Done in f78a6cc6b